backend/devices/bitbox02/device.go

```diff
@@ -391,3 +399,30 @@ func (device *Device) ChannelHashVerify(ok bool) {
 		device.changeStatus(StatusPairingFailed)
 	}
 }
+
+// BTCPub queries the device for a btc, ltc, tbtc, tltc xpub or address.
+func (device *Device) BTCPub(
```

Comment:
the message is called `BTCPubRequest`, so the method name is consistent with that.

`BTC` prefix in the msg is a stand in for btc+btc-like coins, to avoid blowing up the number of messages. Other incompatible coins (or coin classes) would have different prefixes.